org.adempiere.base/src/org/compiere/model/MUserDefProc.java

```diff
@@ -87,7 +87,7 @@ public MUserDefProc(Properties ctx, MUserDefProc copy, String trxName) {
 	private static MUserDefProc[] getAll (Properties ctx, int processID)
 	{
 		if (m_fullList == null) {
-			m_fullList = new Query(ctx, MUserDefProc.Table_Name, "IsActive='Y'", null).list();
+			m_fullList = new Query(ctx, MUserDefProc.Table_Name, "IsActive='Y' AND AD_Client_ID IN (0,?)", null).setParameters(Env.getAD_Client_ID(ctx)).list();
```

Comment:
Hi @igorpojzl 

You must apply here the same approach using m_fullMap as in MUserDefInfo

Regards,

Carlos Ruiz